base.py

```diff
@@ -773,15 +778,18 @@ class MBDynReadLog(bpy.types.Operator):
     bl_label = "MBDyn .log file parsing"
 
     def execute(self, context):
+        mbs = context.scene.mbdyn
         ret_val, obj_names = parse_log_file(context)
 
         missing = context.scene.mbdyn.missing
         if len(obj_names) > 0:
             message = "Some of the nodes/elements are missing in the new .log file"
-            self.report({'WARNING'}, message)
             baseLogger.warning(message)
             hide_or_delete(obj_names, missing)
-            return {'FINISHED'}
+
+        if len(mbs.disabled_output) > 0:
+            message = "No output for nodes " + mbs.disabled_output
+            baseLogger.warning(message)
```

Comment:
@zanoni-mbdyn I'm really at a loss for what's happening here.
If your animation works as expected after disabling a node or more, then that means `len(mbs.disabled_output) > 0`.
If that wasn't the case, your animation wouldn't work.
Could just give me a copy of the log file to check?